docs/standard/microservices-architecture/architect-microservice-container-applications/asynchronous-message-based-communication.md

```diff
@@ -75,7 +75,7 @@ A challenge when implementing an event-driven architecture across multiple micro
 
 -   Using [transaction log mining](http://www.scoop.it/t/sql-server-transaction-log-mining).
 
--   Using full [Event Sourcing](https://msdn.microsoft.com/en-us/library/dn589792.aspx) pattern.
+-   Using full [Event Sourcing](https://msdn.microsoft.com/library/dn589792.aspx) pattern.
```

Comment:
change this to https://docs.microsoft.com/azure/architecture/patterns/event-sourcing